index.html

```diff
@@ -6553,6 +6553,195 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
     -->
     </pre>
   </aside>
+
+  <section class="informative changed"><h3>Property-based data indexing</h3>
+    <p>In its simplest form (as in the examples above),
+      data indexing assigns no semantics to the keys of an <a>index map</a>.
+      However, in some situations,
+      the keys used to index objects are semantically linked to these objects,
+      and should be preserved not only syntactically, but also semantically.
+    </p>
+    <p>If the <a>processing mode</a> is set to <code>json-ld-1.1</code>,
+      <code>"@container": "@index"</code> in a term description can be accompanied with
+      an <code>"@index"</code> key. The value of that key must map to an <a>IRI</a>,
+      which identifies the semantic property linking each object to its key.
+    </p>
+    <aside class="example ds-selector-tabs"
```

Comment:
We should add some description of the restrictions on using property-based indexing, particularly on value objects and the inability to use on graph maps.

Reply:
Fixed in 2ed383d58b4b00589be1e92838c62876e78c5ee6.